src/job/job.spec.js

```diff
@@ -167,4 +167,26 @@ describe("job", () => {
       ).rejects.toMatchSnapshot();
     });
   });
+
+  describe(".runSequence() :", () => {
+    afterEach(async () => {
+      await Promise.all(
+        xo.objects.all[config.vmIdXoTest].snapshots.map(id =>
+          xo
```

Comment:
Here, we are deleting all VM snapshots. We should delete only the snapshot created by the test.